src/rules/unit-blacklist/__tests__/index.js

```diff
@@ -0,0 +1,73 @@
+import {
+  ruleTester,
+  warningFreeBasics,
+} from "../../../testUtils"
+import rule, { ruleName, messages } from ".."
+
+const testRule = ruleTester(rule, ruleName)
+
+testRule([
+  "px",
+  "vmin",
+], tr => {
+  warningFreeBasics(tr)
+
+  tr.ok("a { line-height: 1; }")
+  tr.ok("a { color: #000; }")
+  tr.ok("a { top: 0; left: 0; }")
+  tr.ok("a { font-size: 100%; }")
+  tr.ok("a { line-height: 1.2rem; }")
+
+  tr.ok("/* 100px */", "ignore unit within comments")
```

Comment:
Please put the comment *within a declaration value*, e.g. `a { width: /* 10px */ 1em; }`. Let's make sure that's ok.